webtreemap.js

```diff
@@ -12,11 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+;(function(root, factory) {
+  if (typeof define === 'function' && define.amd)
+    define([], factory);
+  else if (typeof module === 'object' && module.exports)
+    module.exports = factory();
+  else
+    root.appendTreemap = factory();
+}(this, function() {
```

Comment:
Since webtreemap currently assumes usage via browser globals, this allows it to continue to be used that way, but also to be imported as a Node.js-style module or an AMD module, as the commented version of the UMD wrapper @rmmh linked to describes.

You might not see it directly in other codebases as it's commonly added as part of a build process which generates a universal build for browser and other module usage from source which uses a module system, but I wanted to do as little as possible in this PR to make it usable as a module.